droidlet/dashboard/web/src/components/AnnotationComponents/PolygonTool.jsx

```diff
@@ -159,6 +168,13 @@ class PolygonTool extends React.Component {
     this.update();
   }
 
+  // simulates pressing enter on web. Only used for mobile version
+  enterButtonMobile() {
```

Comment:
Nit: rename this to `pressEnterOnMobile` or something that makes it immediately clear what the function does